### PR TITLE
Refactoring backend

### PR DESCRIPTION
**Pull request by sfparrett**

### CODE REVIEW

File: test_social_features.py [hunk 1]
```diff
@@ -3,64 +3,64 @@
 from database import threadDatabase
```
Comment (akwong189) [inline test_social_features.py:4]: is this supposed to be commented out?

**Review by akwong189 — COMMENTED** [review]
test_social_features is commented out? Everything else looks good

File: utils/article.py [hunk 2]
```diff
@@ -21,3 +22,243 @@ def convertor(article):
         if isinstance(article_data, list):
```
Comment (Jack-kelly-22) [inline utils/article.py:22]: nice, using multiple inheritance really helped clean up and declutter the database.py file.
this

File: utils/podcast.py [hunk 3]
```diff
@@ -0,0 +1,189 @@
+import spotipy
+from spotipy.oauth2 import SpotifyOAuth
+import random
+import logging
+
+
+# implement threading for the podcast uploading 
+logger = logging.getLogger("root")
+
+TOPIC_LIST = [
```
Comment (Jack-kelly-22) [inline utils/podcast.py:10]: maybe this data could be loaded from a text file or even just another python file. not a big deal but makes the file a little overwhelming

File: utils/podcast.py [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+                podcast_list[name] = {
+                    "images": item["images"],
+                    "publisher": item["publisher"],
+                    "uri": item["uri"],
+                    "description": item["description"],
+                }
+
+        return podcast_list
+
+    def get_random_indexes(self, count, range):
```
Comment (Jack-kelly-22) [inline utils/podcast.py:131]: Nice Docstring comments!

File: utils/podcast.py [hunk 5]
```diff
@@ -0,0 +1,189 @@
...
+        )
+
+        if len(payload) == 0:
+            return {"message": "no articles possible"}, 404
+
+        else:
+            return payload
+
+
+# social features
```
Comment (Jack-kelly-22) [inline utils/podcast.py:186]: maybe these lines can get the boot

File: utils/social_features.py [hunk 6]
```diff
@@ -0,0 +1,66 @@
+class SocialFeatures:
```
Comment (Jack-kelly-22) [inline utils/social_features.py:1]: Another good use of multiple inheritance.

**Review by Jack-kelly-22 — APPROVED** [review]

**Pull request merged**